launch/gz_intel_realsense_d435.launch.py

```diff
@@ -32,19 +32,19 @@ def generate_launch_description():
     )
 
     robot_namespace = LaunchConfiguration("robot_namespace")
-    device_namespace = LaunchConfiguration("device_namespace")
+    component_name = LaunchConfiguration("component_name")
     gz_bridge_name = LaunchConfiguration("gz_bridge_name")
 
     namespaced_gz_bridge_config_path = ReplaceString(
         source_file=gz_bridge_config_path,
         replacements={
             "<robot_namespace>": robot_namespace,
-            "<device_namespace>": device_namespace,
+            "<component_name>": component_name,
         },
     )
 
     declare_device_namespace = DeclareLaunchArgument(
```

Comment:
```suggestion
    declare_component_name = DeclareLaunchArgument(
```
Applies to all launch files

Reply:
Replaced